handler.js

```diff
@@ -358,42 +358,92 @@ class GithubBuild {
     ]
   }
 
+  getForceCommand() {
+    // parse [force uat|functional <argument1> ... <argumentN>]
+    const commandRegex = new RegExp(/\[force (uat|functional)(\s[^\]]+)?]/i);
+    let matches = commandRegex.exec(this.getCommitMsg());
+
+    if (matches === null || matches.length < 3) {
+      return null;
+    }
+
+    return [
+      // matched "uat" or "functional"
+      matches[1],
+
+      // matched argument
+      matches[2]
+    ];
+  }
+
   getTests() {
     let tests = [];
     const branch = this.getBranch();
-    const forceRegex = new RegExp(/.*\[force ([a-z]+)([\s][^\]]+)?].*/gm);
-    const commitMsg = this.getCommitMsg();
-    const matches = forceRegex.exec(commitMsg);
-    console.log(matches);
-    let forceCommand = null;
-    let forceTest = null;
-    if (matches !== null && matches.length >= 2 && ('push' === this.getEventType() || 'api' === this.getEventType())) {
-      forceCommand = matches[1];
-      forceTest = matches.length >= 3 ? matches[2] : null;
-    }
-    console.log(forceCommand, forceTest);
 
-    tests.push({
-      name: "js-php",
-      type: "unit-tests",
-      deployable: false
-    });
+    // deploy and run all tests on release and master
+    // ignore any commands we get
+    if(branch === "release" || branch === "master") {
```

Comment:
Do we need to run all tests on master @mpgo13?

Reply:
Yes, for code freeze and nightly build we would run full UAT and completely ignore all commands @tuanmh

Reply:
freeze should be to release right? nightly build, we can provide custom data to CodeBuild as we do atm. This would run UAT on every commit to master, wouldn't we want it?

Reply:
Oh yes true! Release should be enough then.